sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala

```diff
@@ -856,11 +856,11 @@ class JDBCV2Suite extends QueryTest with SharedSparkSession with ExplainSuiteHel
     val df11 = sql(
       """
         |SELECT * FROM h2.test.employee
-        |WHERE GREATEST(bonus, 1100) > 1200 AND LEAST(salary, 10000) > 9000 AND RAND(1) < 1
```

Comment:
We already test LEAST at https://github.com/apache/spark/blob/c11fc3005d5407ca44c848de7d074d2a10f743a4/sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala#L869